pkg/task/restore.go

```diff
@@ -86,9 +88,17 @@ func (cfg *RestoreConfig) ParseFromFlags(flags *pflag.FlagSet) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+
 	if cfg.Config.Concurrency == 0 {
 		cfg.Config.Concurrency = defaultRestoreConcurrency
 	}
+	if cfg.Config.Concurrency < defaultRestoreConcurrency {
```

Comment:
This sounds too strict. The program will warn you if the concurrency is less than 128, while what this PR prevents is setting the concurrency to 4. The warning limit could be somewhere between 4 and 128.

Reply:
I didn't get the point why this PR prevents setting the concurrency to 4?
I think the concurrency lower than `defaultRestoreConcurrency`(128) is not suitable.

Reply:
The question is not about "preventing" the concurrency to be 4, it's having the warning when I set concurrency to 40. 

The problem is we're using the same concurrency flag to both BatchSplit and the workerPool for file.Import.

Reply:
what about just set lower bound of concurrency to 128?